### PR TITLE
fix migrate enable to actually enable the job runner 

### PR DESCRIPTION
**Pull request by malaffoon**
Missed this corner case: after using the actuator end-point to enable migrate, triggering a migrate using the actuator end-point doesn't work.

### CODE REVIEW

File: migrate-common/src/test/java/org/opentestsystem/rdw/migrate/common/MigrateEndpointTest.java [hunk 1]
```diff
@@ -41,7 +41,7 @@ public void itShouldGetGoodMigrateInfo() {
     }
 
     @Test
-    public void itShouldBadGoodMigrateInfo() {
+    public void itShouldGetBadMigrateInfo() {
```
Comment (malaffoon) [inline migrate-common/src/test/java/org/opentestsystem/rdw/migrate/common/MigrateEndpointTest.java]: unrelated: fix test name
